src/include/distributed/intermediate_result_pruning.h

```diff
@@ -23,6 +23,9 @@ extern List * FindAllWorkerNodesUsingSubplan(HTAB *intermediateResultsHash,
 extern HTAB * MakeIntermediateResultHTAB(void);
 extern void RecordSubplanExecutionsOnNodes(HTAB *intermediateResultsHash,
 										   DistributedPlan *distributedPlan);
+extern IntermediateResultsHashEntry * SearchIntermediateResult(HTAB
+															   *intermediateResultsHash,
```

Comment:
nit; I like keeping the types and variable names on the same line. Doesn't fit on the line above; you can move `HTAB` to  this line.